bin.js

```diff
@@ -121,7 +121,8 @@ cli
   .action(registerSpace)
 
 cli
-  .command('space add <proof>')
+  .command('space add [proof]')
+  .option('--base64', 'provide proof as base64 encoded string')
```

Comment:
IMO it would be helpful for noobs to know more about what is expected as the bytes that are base64d. iirc the idea is we accept any UCAN serialization? but esp car files?
(e.g. do you want base64(cidOfProofCar) or base64(proofCarWithDagUcanCbor)) ?